WordPressAuthenticatorTests/GoogleSignIn/Data+Base64URLTests.swift

```diff
@@ -37,4 +39,45 @@ class DataBase64URLDecoding: XCTestCase {
             Data()
         )
     }
+
+    // MARK: - Encoding
+
+    func testBase64URLEncoding() {
+        XCTAssertEqual(
+            "hello world".data(using: .utf8)!.base64URLEncodedString(),
```

Comment:
Same note about string -> data here, unless you want to deal with optionality, in which case I'd suggest `XCTUnwrap`?

Reply:
Excellent, thanks!

See caa86091